core/interfaces/__init__.py

```diff
@@ -31,6 +31,7 @@ def run_async(self, source, target, reverse):
         to_load = target or self.get_next(source, reverse)
         if not to_load:
             return
+        to_load = "tag" if to_load == "tags" else to_load
```

Comment:
It is mostly for backward compatibility. Some uses may have specified the old `tags` in the order. I agree that it’s ugly...May be just put a comment saying that it’s for backward compatibility?

Reply:
I recommend that you change the mappings then. E.g 

```
    view_signatures = {
        "status": "git_savvy.status_view",
        "branch": "git_savvy.branch_view",
        "rebase": "git_savvy.rebase_view",
        "tag": "git_savvy.tags_view",
        "tags": "git_savvy.tags_view",  # for backwards compatibility
        "graph": "git_savvy.log_graph_view"
    }

```

But now you also need a deprecation warning for the user bc otherwise this flag stays forever. 

The intent of this PR is to make something easier but as it stands it makes the program more complicated. That's the problem. 

I just do the review here, but I would revert and only change the user facing command caption.

Reply:
I actually wouldn't even rename the caption, honestly. The command is called `gs_show_tags` and what the user sees is 'tags'. 

It is more like 'branch' should be named 'branches'.

Reply:
Anyway, they should be consistent. I vote for `git: tag` for now.